### DISCUSSION TITLE
Parte 2: Uso de librería uthash.h

### DISCUSSION THREAD

**Discussion by MatiLaB** [IIC2133-PUC/2024-2#163] [Tarea 2] [answered]
Hola, estaba revisando como hacer la parte 2 (un poco tarde lo sé) y encontré la librería uthash.h que al parecer sirve para manejar tablas de hash. Mi duda es que si se puede usar esta librería o se busca que nosotros definamos y construyamos a mano nuestras tablas hash. No he revisado si se puede usar con una función propio para el valor de hash, pero quería consultar antes de seguir investigando.

**Comment by sebacarrascop** [comment, marked as answer]
hola! la idea es que ustedes manejen el hashing.
